### PR TITLE
Add requirements files for installing providers and provisioners for devel

### PR DESCRIPTION
**Pull request by rcarrillocruz**
This is for convenience.

### CODE REVIEW

**Comment by rcarrillocruz** [comment]
recheck

File: providers_from_galaxy.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+- ansible-network.aws
+- ansible-network.cisco_ios
+- ansible-network.vyos
+- ansible-network.rhel
```
Comment (pabelanger) [inline providers_from_galaxy.yaml]: This looks to be missing src: prefix.

**Review by pabelanger — COMMENTED** [review]
I think there is an issue with one of the requirements file

File: defaults/main.yaml [hunk 2]
```diff
@@ -1,4 +1,10 @@
 ---
+cloud_vpn_install_pip_requirements: false
+cloud_vpn_install_providers: false
+cloud_vpn_install_provisioners: false
+cloud_vpn_providers_file: "{{ role_path }}/providers_from_devel.yaml"
+cloud_vpn_provisioners_file: "{{ role_path }}/provisioners_from_devel.yaml"
```
Comment (ganeshrn) [inline defaults/main.yaml]: Shouldn't this default to installing from the galaxy in the released version of the role as the devel branch for provider roles might unintentionally break some functionality when using a released version of `cloud_vpn` role.

Comment (trishnaguha) [reply defaults/main.yaml]: +1

**Review by trishnaguha — APPROVED** [review]

**Review by ganeshrn — APPROVED** [review]

**Pull request merged**